src/App.vue

```diff
@@ -65,18 +65,23 @@
         Default styles should be for vertical mobile devices
         (devices narrower than --ns-breakpoint-mobile)
 
-        Small devices (landscape phones)
-        @media (min-width: var(--ns-breakpoint-mobile)) { ... }
+        // Small devices (landscape phones)
```

Comment:
I'm attaching a screenshot.  It doesn't seem like the use of mobile phone real estate is optimized yet.  At least in this screenshot, there's clearly room to wrap the text but it isn't, it runs off the right, as does the picture.

![Screenshot from 2024-07-05 11-38-51](https://github.com/numberscope/frontscope/assets/15117097/e8f3c50e-ee9d-4d28-bcfd-f3bee9657793)

Reply:
Wow, ugh, yes, it does seem like this needs to be addressed before merging this PR. Not really my forte, so if anyone else wants to jump in I would welcome it, but if this PR ends up stuck on just this I will give it a shot.